### ISSUE TITLE
Take http status code as a configuration value

### ISSUE THREAD

**Issue by freman** [BTBurke/caddy-jwt#55]
Hi,

We would like to be able to configure the status code, specifically instead of 303 we want to use 308

Maybe a config property `RedirectCode=308` or have a `,308` after the url?

**Comment by magikstm** [comment]
Are you using caddy-jwt by itself or with another plugin?

**Comment by freman** [comment]
by itself, it redirects to our auth server (I mean with a whole bunch of plugins in caddy but not generally *with* caddy-jwt)

**Comment by BTBurke** [comment]
I don't think that is going to happen.  Caddy 2 will be coming out in a few months and all of the existing plugins will have to be rewritten to work with the new module system.  I haven't decided yet whether I'll port the JWT plugin or let someone else take over, but with the new design there should be more flexibility to decide how you want it to redirect.

**Comment by freman** [comment]
Aww, but I already did it :D
